cli/flox/src/commands/show.rs

```diff
@@ -126,7 +137,70 @@ fn construct_show_params(
     Ok(search_params)
 }
 
-fn render_show(search_results: &[SearchResult]) -> Result<()> {
+fn render_show_catalog(
+    search_results: &[SearchResult],
+    expected_systems: &HashSet<System>,
+) -> Result<()> {
+    if search_results.is_empty() {
+        // This should never happen since we've already checked that the
+        // set of results is non-empty.
+        bail!("no packages found");
+    }
+    let pkg_name = search_results[0].rel_path.join(".");
+    let description = search_results[0]
+        .description
+        .as_ref()
+        .map(|d| d.replace('\n', " "))
+        .unwrap_or(DEFAULT_DESCRIPTION.into());
+    println!("{pkg_name} - {description}");
+
+    // Organize the versions to be queried and printed
+    let version_to_systems = {
+        let mut map = BTreeMap::new();
+        for pkg in search_results.iter() {
+            if let Some(ref version) = pkg.version {
+                map.entry(version.clone())
+                    .or_insert(HashSet::new())
+                    .insert(pkg.system.clone());
+                // map.insert(version.clone(), pkg.system.clone());
```

Comment:
```suggestion
```

Reply:
I'll pick that up in the resolution messages PR